### DISCUSSION TITLE
Perguntas

### DISCUSSION THREAD

**Discussion by AmandaThayna** [AmandaThayna/devamanda#2] [Q&A] [answered]
Tenha duvidas em Back-end, realizar codigo de pix

**Comment by rehnata2** [comment, marked as answer]
Você pode realizar o pix no gerarpix e utilizar o canva

**Comment by AmandaThayna** [comment]


**Reply by rehnata2** [reply]
Java é uma linguagem de programação OOP .
Javascript é uma língua de script OOP
